readme.md

```diff
@@ -243,6 +243,7 @@ Most of these are actions built into REAPER, but a few are very useful actions f
 #### Zoom
 - Zoom out horizontal: - or NumPad-
 - Zoom in horizontal: = or NumPad+
+- Set horizontal zoom to default project setting
```

Comment:
All of these were missing the "View: " part of the action name, so I'm adding it to all of them here.
```suggestion
- View: Zoom out horizontal: - or NumPad-
- View: Zoom in horizontal: = or NumPad+
- View: Set horizontal zoom to default project setting
```